src/style.css

```
@@ -0,0 +1,110 @@
+body {
+  margin: 0;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+.card {
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+}

```

```suggestion
.card {
  padding: 16px;
  display: flex;
  flex-direction: column;
  max-width: 200px;
  border: 1px solid #f3f3f3;
  border-radius: 5px;
}

```

white background is default